### PR TITLE
Watchlist template queries

### PR DESCRIPTION
**Pull request by petebryan**
Updated queries looking at unusual user query activity to also include queries that use the new Watchlist templates.

### CODE REVIEW

File: Hunting Queries/LAQueryLogs/NewUserCallingSensitiveWatchlist.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  | where QueryText has_any ('_GetWatchlist("VIPUsers")', "_GetWatchlist('VIPUsers')", '_GetWatchlist("ServiceAccounts")', "_GetWatchlist('ServiceAccounts')",'_GetWatchlist("HighValueAssets")', "_GetWatchlist('HighValueAssets')")
+  | summarize by AADEmail
+  | join kind = rightanti (LAQueryLogs
+  | where TimeGenerated between(starttime..endtime)
+  | where QueryText has_any ('_GetWatchlist("VIPUsers")', "_GetWatchlist('VIPUsers')", '_GetWatchlist("ServiceAccounts")', "_GetWatchlist('ServiceAccounts')",'_GetWatchlist("HighValueAssets")', "_GetWatchlist('HighValueAssets')"))
+  on AADEmail
+  | project TimeGenerated, AADEmail, QueryText, RequestClientApp, RequestTarget
+  | extend timestamp = TimeGenerated, AccountCustomEntity = AADEmail
+
+entityMappings:
```
Comment (shainw) [inline Hunting Queries/LAQueryLogs/NewUserCallingSensitiveWatchlist.yaml:30]: RequestTarget is an Azure Resource, can we add this to the entity mapping - https://docs.microsoft.com/en-us/azure/sentinel/entities-reference#azure-resource

File: Detections/LAQueryLogs/UserSearchingForVIPUserActivity.yaml [hunk 2]
```diff
@@ -20,8 +20,10 @@ relevantTechniques:
 query: |
   // Replace these with the username or emails of your VIP users you wish to monitor for.
   let vips = dynamic(['vip1@email.com','vip2@email.com']);
+  // Add users who are allowed to conduct these searches - this could be specific SOC team members
+  let allowed_users = dynamic([]);
   LAQueryLogs
-  | where QueryText has_any (vips)
+  | where QueryText has_any (vips) or QueryText has_any ('_GetWatchlist("VIPUsers")', "_GetWatchlist('VIPUsers')")
   | project TimeGenerated, AADEmail, RequestClientApp, QueryText, ResponseRowCount, RequestTarget
```
Comment (shainw) [inline Detections/LAQueryLogs/UserSearchingForVIPUserActivity.yaml:27]: RequestTarget is an Azure Resource, can we add this to the entity mapping - https://docs.microsoft.com/en-us/azure/sentinel/entities-reference#azure-resource.  Also, RequestClientApp might fit in Name under [Cloud Application](https://docs.microsoft.com/en-us/azure/sentinel/entities-reference#cloud-application)

**Review by shainw — DISMISSED** [review]
One non-blocking suggestions.

File: Detections/LAQueryLogs/UserSearchingForVIPUserActivity.yaml (hunk 2 above)
Comment (petebryan) [reply Detections/LAQueryLogs/UserSearchingForVIPUserActivity.yaml:27]: Done

File: Hunting Queries/LAQueryLogs/NewUserCallingSensitiveWatchlist.yaml (hunk 1 above)
Comment (petebryan) [reply Hunting Queries/LAQueryLogs/NewUserCallingSensitiveWatchlist.yaml:30]: Done

**Review by shainw — APPROVED** [review]

**Pull request merged**